LBHackney-IT/lbh-fss-portal-frontend
# Feature/accordion-sidebar

**Description**

Desired functionality is for navigation sidebar to switch to accordion when viewed on mobile. 

**Test instructions** 

- pull this branch 
- navigate to **/organisations/9999/edit** and check navigation sidebar switches to accordion when viewed on mobile 
- navigate to **/services/15971/edit** and check navigation sidebar switches to accordion when viewed on mobile 

![image](https://user-images.githubusercontent.com/69238887/95370331-81de9580-08d0-11eb-9cee-0234d1a311d7.png)


**Ticket**
https://app.clickup.com/t/88wwvq
